src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java

```diff
@@ -674,12 +674,12 @@ void setBlock_ChangeBlock()
 		Block block = world.getBlockAt(0, 1, 0);
 		BlockData data = block.getBlockData();
 		assertEquals(data.getMaterial(), world.getBlockData(location).getMaterial());
-		BlockDataMock mock = new BlockDataMock(Material.GRASS);
+		BlockDataMock mock = new BlockDataMock(Material.SHORT_GRASS);
```

Comment:
And here

Reply:
For how this test looked initially, one of the materials was Material.GRASS, and another Material.GRASS_BLOCK. By just thinking quickly about this, it seemed like substituting Material.GRASS -> Material.SHORT_GRASS was a valid choice.